README.md

```diff
@@ -24,11 +24,17 @@ python3 -m http.server 4321
 ### Build Docker Image
 
 ```sh
-docker build -t oncokb/oncokb-sop:0.0.1 .
+docker build --platform=linux/amd64 -t oncokb/oncokb-sop:x.x.x .
```

Comment:
We should also build for both amd64 and arm (https://www.docker.com/blog/multi-arch-images/)
We currently have amd64 AWS machines, but eventually want to use arm machines.

Reply:
I missed this... I don't think we really need to do this since we have no other users of our sop image.

Once we're ready for arm it's pretty quick for start building for just arm.